x-pack/plugins/maps/public/classes/styles/vector/components/label/vector_style_label_border_size_editor.js

```diff
@@ -53,10 +53,7 @@ export function VectorStyleLabelBorderSizeEditor({
   }
 
   const labelBorderSizeForm = (
-    <EuiFormRow
-      label={getVectorStyleLabel(VECTOR_STYLES.LABEL_BORDER_SIZE)}
-      display="columnCompressed"
-    >
+    <EuiFormRow label={getVectorStyleLabel(VECTOR_STYLES.LABEL_BORDER_SIZE)}>
```

Comment:
Yes

There were only 2 properties that used `columnCompressed`. The problem with column compressed is that it puts 2 types of layout in the form

Without column compressed
```
Property title
input
```

With column compressed.
```
Property title.  input
```

The form is very long. My eyes kept getting tripped up on column compressed properties because they did not follow the pattern of the others.

This is a person preference that can be remove from this PR to debate separately. I do not feel that strongly about it.

Reply:
The layer style panel is getting quite busy and crowded, I'd also open a separate discussion on ways to make it easier to digest

Reply:
Created https://github.com/elastic/kibana/issues/146060 to track